### ISSUE TITLE
[v2.0] Error trying to add package

### ISSUE THREAD

**Issue by flippyhead** [alanning/meteor-roles#189]
I didn't see that v2 was published to atmosphere yet so I git cloned the 2.0 branch to my PACKAGES_DIR and added, but got the below error:

```
meteor add alanning:roles
 => Errors while adding packages:             
                                              
While building package alanning:roles:
error: Symlink cycle detected at examples/flow-router-advanced/packages/alanning:roles
```

Any suggestions?

Thanks and really looking forward to 2.0!

**Comment by alanning** [comment]
Sorry about this.  Another user [reported](https://github.com/alanning/meteor-roles/issues/171) that removing the `examples` folder fixes this.

Won't be an issue once v2.0 is officially published to atmosphere.

**Comment by flippyhead** [comment]
Roger, just discovered this myself. Thanks!

**Comment by mitar** [comment]
I discovered that this is a regression in Meteor 1.3 in comparison with 1.2 and previous versions: https://github.com/meteor/meteor/issues/6665

**Comment by mitar** [comment]
So we might have to change the pattern we use in examples to use a symlink.

**Comment by mitar** [comment]
This should be fixed with Meteor 1.3.2. Please report if it does not work.
